pkgs/by-name/wi/wideriver/package.nix

```
@@ -0,0 +1,45 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  wayland,
+  wayland-scanner,
+  pkg-config,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wideriver";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "alex-courtis";
+    repo = "wideriver";
+    rev = version;
```

adding `refs/tags/` removes ambiguity
and use `finalAttrs` to get version
```suggestion
    rev = "refs/tags/${finalAttrs.version}";
```